src/domain/core/src/services/transform_service.rs

```diff
@@ -133,6 +133,12 @@ pub enum TransformError {
         AccessError,
     ),
     #[error(transparent)]
+    InvalidInterval(
+        #[from]
+        #[backtrace]
+        InvalidIntervalError,
+    ),
```

Comment:
Q: don't really understand the difference with `Internal` -- maybe use a more specific name?

Reply:
"invalid interval" is very clear, there is no chain path from the end back to start.

Reply:
oh, yes --  I read it earlier as `InvalidInter`-N-`al`